rules/sample_whatshap.smk

```diff
@@ -79,10 +79,12 @@ rule whatshap_haplotag:
     log: f"samples/{sample}/logs/whatshap/haplotag/{sample}.{ref}.{{movie}}.log"
     benchmark: f"samples/{sample}/benchmarks/whatshap/haplotag/{sample}.{ref}.{{movie}}.tsv"
     params: "--tag-supplementary"
+	threads: 4
     conda: "envs/whatshap.yaml"
     shell:
         """
         (whatshap haplotag {params} \
```

Comment:
Will snakemake variable substitution replace {params} with "--tag-supplementary 4" because of the mix of named and unnamed params?